config.md

```diff
@@ -324,6 +324,7 @@ Presently there are `Prestart`, `Poststart` and `Poststop`.
 Hooks allow one to run programs before/after various lifecycle events of the container.
 Hooks MUST be called in the listed order.
 The state of the container is passed to the hooks over stdin, so the hooks could get the information they need to do their work.
+The container state data format is defined by various runtime implementations, and is not included in this specification.
```

Comment:
Ouch, my mistake. I searched the https://github.com/opencontainers/runtime-spec/blob/master/specs-go/config.go and didn't find the hook state definition, so I thought it would be a runtime specific field and we never defined this in specs, it turns out to be here: https://github.com/opencontainers/runtime-spec/blob/master/specs-go/state.go

I'll revoke this and send an opposite PR to add definition of Hook State in this spec later.

Sorry for my misread~